demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -103,53 +122,95 @@ def execute_update(self):
-                                f"created release notes found at {rn_path} and document any changes you "
-                                f"made by replacing '%%UPDATE_RN%%'.\n - Commit "
-                                f"the new release notes to your branch.\nFor information regarding proper"
-                                f" format of the release notes, please refer to "
-                                f"https://xsoar.pan.dev/docs/integrations/changelog", LOG_COLORS.GREEN)
-                    return True
-                else:
-                    click.secho("No changes to pack files were detected from the previous time "
-                                "this command was run. The release notes have not been "
-                                "changed.", fg='green')
+            return self.create_pack_rn(rn_path, changed_files, new_metadata, docker_image_name)
+
+    def create_pack_rn(self, rn_path: str, changed_files: dict, new_metadata: dict,
+                       docker_image_name: Optional[str]) -> bool:
+        """ Checks whether the pack requires a new rn and if so, creates it.
+
+            :param
+                rn_path: The rn path
+                changed_files: The changed files details
+                new_metadata: The new pack metadata
+                docker_image_name: The docker image name
+
+
+            :rtype: ``bool``
+            :return
+                Whether the RN was updated successfully or not
+        """
+        rn_string = self.handle_existing_rn_version_path(rn_path)
+        if not rn_string:
+            rn_string = self.build_rn_template(changed_files)
+        if len(rn_string) > 0 or self.is_force:
+            if self.is_bump_required():
+                self.write_metadata_to_file(new_metadata)
+            self.create_markdown(rn_path, rn_string, changed_files)
+            try:
+                run_command(f'git add {rn_path}', exit_on_error=False)
+            except RuntimeError:
+                print_warning(f'Could not add the release note files to git: {rn_path}')
+            if self.existing_rn_changed:
+                print_color(f"Finished updating release notes for {self.pack}."
+                            f"\nNext Steps:\n - Please review the "
+                            f"created release notes found at {rn_path} and document any changes you "
+                            f"made by replacing '%%UPDATE_RN%%'.\n - Commit "
+                            f"the new release notes to your branch.\nFor information regarding proper"
+                            f" format of the release notes, please refer to "
+                            f"https://xsoar.pan.dev/docs/integrations/changelog", LOG_COLORS.GREEN)
+                return True
             else:
-                click.secho("No changes which would belong in release notes were detected.", fg='yellow')
+                click.secho(f"No changes to {self.pack} pack files were detected from the previous time "
```

Comment:
while we already changing this, be consistent and either use `print_color` or `click.secho` but not both.

Reply:
Done